resources/META-INF/plugin.xml

```diff
@@ -256,6 +256,14 @@
               description="Restart"
               icon="AllIcons.Actions.Restart">
       </action>
+      <action id="Flutter.Menu.RunProfileAction" class="io.flutter.actions.RunProfileFlutterApp"
+              description="Profile"
+              icon="FlutterIcons.RunProfile">
+      </action>
+      <action id="Flutter.Menu.RunReleaseAction" class="io.flutter.actions.RunReleaseFlutterApp"
+              description="Release"
+              icon="FlutterIcons.RunRelease">
```

Comment:
That's what I started with actually and then convinced myself we might want to swap this out.  Anyway, I'm back to agreeing. 👍